src/Sylius/Behat/Context/Transform/AddressContext.php

```diff
@@ -99,79 +109,59 @@ public function createEmptyAddress()
 
     /**
      * @Transform /^address for "([^"]+)" from "([^"]+)", "([^"]+)", "([^"]+)", "([^"]+)", "([^"]+)"$/
-     * @Transform /^"([^"]+)" addressed it to "([^"]+)", "([^"]+)" "([^"]+)" in the "([^"]+)"(?:|, "([^"]+)")$/
-     * @Transform /^of "([^"]+)" in the "([^"]+)", "([^"]+)" "([^"]+)", "([^"]+)"(?:|, "([^"]+)")$/
-     * @Transform /^addressed it to "([^"]+)", "([^"]+)", "([^"]+)" "([^"]+)" in the "([^"]+)"(?:|, "([^"]+)")$/
-     * @Transform /^address (?:|is )"([^"]+)", "([^"]+)", "([^"]+)", "([^"]+)", "([^"]+)"(?:|, "([^"]+)")$/
-     * @Transform /^address as "([^"]+)", "([^"]+)", "([^"]+)", "([^"]+)", "([^"]+)"(?:|, "([^"]+)")$/
+     * @Transform /^"([^"]+)" addressed it to "([^"]+)", "([^"]+)" "([^"]+)" in the "([^"]+)", "([^"]+)"$/
+     * @Transform /^of "([^"]+)" in the "([^"]+)", "([^"]+)" "([^"]+)", "([^"]+)", "([^"]+)"$/
+     * @Transform /^addressed it to "([^"]+)", "([^"]+)", "([^"]+)" "([^"]+)" in the "([^"]+)", "([^"]+)"$/
+     * @Transform /^address (?:|is |as )"([^"]+)", "([^"]+)", "([^"]+)", "([^"]+)", "([^"]+)", "([^"]+)"$/
      */
-    public function createNewAddressWithName($name, $street, $postcode, $city, $countryName, $provinceName = null)
+    public function createNewAddressWithNameAndProvince($name, $street, $postcode, $city, $countryName, $provinceName)
     {
-        $countryCode = $this->countryNameConverter->convertToCode($countryName);
         list($firstName, $lastName) = explode(' ', $name);
 
-        return $this->createAddress($countryCode, $firstName, $lastName, $city, $street, $postcode, $provinceName);
+        return $this->exampleAddressFactory->create([
+            'country_code' => $this->countryNameConverter->convertToCode($countryName),
+            'first_name' => $firstName,
+            'last_name' => $lastName,
+            'company' => null,
+            'customer' => null,
+            'phone_number' => null,
```

Comment:
Do we need to pass `null` for optional fields like `phone_number` or `company`? Can't we just skip them?